content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -226,6 +226,64 @@ As a client, you can request `BOOKMARK` events by setting the
 assume bookmarks are returned at any specific interval, nor can clients assume that
 the API server will send any `BOOKMARK` event even when requested.
 
+## Streaming lists
+
+{{< feature-state for_k8s_version="v1.27" state="alpha" >}}
+
+On large clusters, retrieving the collection of some resource types may result in
+a significant increase of resource usage (primarily RAM) on the control plane.
+In order to alleviate its impact and simplify the user experience of the **list** + **watch**
+pattern, Kubernetes v1.27 introduces as an alpha feature the support
+for requesting the initial state (previously requested via the **list** request) as part of
+the **watch** request.
+
+Provided that the `WatchList` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+is enabled, this can be achieved by specifying `sendInitialEvents=true` as query string parameter
```

Comment:
The 1.27 version is mentioned 2 lines above (in addition to the header of the section).  I don't think it makes sense to constantly repeat it - for me as a reader it would be more annoying than useful.

Reply:
I'm proposing that _this_ (and the feature state shortcode, which lots of readers don't fully notice) is where we make the requirements clear.

That way, we're documenting in one place both:
- the version requirement
- the need to explicitly enable a feature gate

My experience suggests putting these details in 2 places is likely to lead to more GitHub issues telling us the docs are wrong (even when the docs are strictly correct and have been misunderstood).